app/services/foreman/parameters/validator.rb

```diff
@@ -13,6 +13,10 @@ def validate!
           validate_regexp
         when "list", "array"
           validate_list
+        when "hash"
```

Comment:
![screen shot 2015-11-09 at 5 26 34 pm](https://cloud.githubusercontent.com/assets/706359/11037576/1402bdfc-8707-11e5-98ea-9048a9e0936d.png)
this is how it currently looks (after two invalid submissions)